fusor-ember-cli/app/adapters/application.js

```diff
@@ -9,6 +10,17 @@ export default DS.ActiveModelAdapter.extend({
     },
     shouldReloadRecord(store, ticketSnapshot) {
       return true;
+    },
+    handleResponse(status /*, headers, payload */) {
+        if(status === 401) {
+            App.EventBus.trigger('displayErrorModal', {
```

Comment:
@eriknelson, PR looks good. 
Not sure we can assume that any 401 response is a timeout?